CMakeLists.txt

```diff
@@ -226,6 +226,7 @@ find_package (mongoc-1.0 1.17.4 EXACT)
 # Static libs common to contextBroker and unitTest binaries
 SET (COMMON_STATIC_LIBS
     microhttpd.a
+    mosquitto.a
```

Comment:
Not using static linking makes RPM stuff harder (as the RPM would need libmosquitto as dependency and it seems it is not available at yum)

```
[fermin@centocho fiware-orion]$ cat /etc/redhat-release 
CentOS Linux release 8.3.2011
[fermin@centocho fiware-orion]$ sudo yum search mosquitto
Última comprobación de caducidad de metadatos hecha hace 0:12:39, el lun 09 ago 2021 07:25:55 EDT.
No se encontraron coincidencias.
```

Reply:
From https://pkgs.org/search/?q=mosquitto

![imagen](https://user-images.githubusercontent.com/1534240/128700651-a55e5158-60a2-4db3-ae85-fdb43dd6f43d.png)

So maybe we could get and old version of mosquitto (via EPEL), but better to use the newest one (2.0.11)

Reply:
NTC (informative)